charts/rancher-gatekeeper/rancher-gatekeeper/100.0.0+up3.5.1/values.yaml

```diff
@@ -1,23 +1,37 @@
 replicas: 3
 auditInterval: 300
+auditMatchKindOnly: false
 constraintViolationsLimit: 20
 auditFromCache: false
 disableValidatingWebhook: false
 validatingWebhookTimeoutSeconds: 3
 enableDeleteOperations: false
+experimentalEnableMutation: false
 auditChunkSize: 0
 logLevel: INFO
+logDenies: false
 emitAdmissionEvents: false
 emitAuditEvents: false
+postInstall:
+  labelNamespace:
+    enabled: true
+    image:
+      repository: line/kubectl-kustomize
```

Comment:
Needs to be mirrored

Reply:
Instead of adding a new mirrored image I changed this to use the `rancher/kubectl` image (and tested it)